docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -122,63 +122,166 @@ class Sidebar extends React.Component<any, any> {
     })
 */
 
+  getActiveCategoryIndex(at: String, sections: ShorthandValue<any>[]) {
+    let i
+    for (i = 0; i < sections.length; i++) {
+      const category = sections[i]
+      if (!('items' in category)) {
+        continue
+      }
+      if (category.items.filter(c => 'title' in c).some(e => e.title.to === at)) {
+        return i
+      }
+    }
+    return -1
+  }
+
+  keyDownCallback(e) {
+    if (e.key !== 'Enter') {
```

Comment:
We are using the `keyboard-key` already. Can we refactor this condition using it?
`if (keyboardKey.getCode(e) === keyboardKey.Enter)`